src/components/UserStory.jsx

```diff
@@ -21,6 +21,22 @@ const titles = {
   team_members: 'Team Members',
   teams: 'Team',
   version_control_systems: 'Version Control System',
+  build_tools: 'Build Tools',
+  community_supports: 'Community Support',
+  company: 'Company',
+  company_website: 'Company Website',
+  funded_by: 'Funded By',
+  industries: 'Industries',
+  organization: 'Organization',
+  platforms: 'Platform',
+  plugins: 'Custom Plugins',
+  programming_languages: 'Programming Languages',
+  project_funding: 'Project Funding',
+  project_website: 'Project Website',
+  summary: 'Summary',
+  team_members: 'Team Members',
+  teams: 'Team',
+  version_control_systems: 'Version Control System',
```

Comment:
```suggestion
```

Reply:
It appears that they are already present in this same file in L8-23. Is there any reason for these duplicates?

Reply:
Could you please commit this change @biru-codeastromer?